### PR TITLE
Removing about-sharesight 

### PR DESCRIPTION
**Pull request by maisiembleton**
The vimaly ticket for the about sharesight page can be found [here](https://vimaly.com/#j8mqm/t/www_Rebuild_About_Sharesight/SEHU6Y6s1ge6UVsBc). 

Removing the about-sharesight page from this repo as it has been created and deployed in the gatsby repo!

### CODE REVIEW

File: spec/helpers/middleman/page_spec.rb [hunk 1]
```diff
@@ -174,7 +174,7 @@
       expect(@app.locale_page(page: 'index')[:page]).to eq('index')
       expect(@app.locale_page(page: 'blog')[:page]).to eq('blog')
       expect(@app.locale_page(page: 'pro')[:page]).to eq('pro')
-      expect(@app.locale_page(page: 'about-sharesight')[:page]).to eq('about-sharesight') # landing page
+      expect(@app.locale_page(page: 'pricing')[:page]).to eq('pricing') # landing page
```
Comment (kylorhall) [inline spec/helpers/middleman/page_spec.rb:177]: Weird, this isn't (and previously wasn't) a landing page…but I'll ignore it, I don't think I really care about test coverage in here as long as it's not breaking.

**Review by kylorhall — APPROVED** [review]
I don't have anything to add, assuming the metadata was copied over correctly…!

**Review by jenniferexong — APPROVED** [review]
Looks good to mee! Hopefully tests will pass now 🤞🏼

File: spec/helpers/middleman/page_spec.rb [hunk 2]
```diff
@@ -187,12 +187,12 @@
...
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('nz'))[:page_title]).to eq('About Us | Sharesight New Zealand')
-      expect(@app.locale_page(page: 'about-sharesight', locale_obj: @app.get_locale_obj('uk'))[:page_title]).to eq('About Us | Sharesight UK')
+      expect(@app.locale_page(page: 'pricing')[:page_title]).to eq('About Us | Sharesight')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('global'))[:page_title]).to eq('About Us | Sharesight')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('au'))[:page_title]).to eq('About Us | Sharesight Australia')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('ca'))[:page_title]).to eq('About Us | Sharesight Canada')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('nz'))[:page_title]).to eq('About Us | Sharesight New Zealand')
+      expect(@app.locale_page(page: 'pricing', locale_obj: @app.get_locale_obj('uk'))[:page_title]).to eq('About Us | Sharesight UK')
```
Comment (jenniferexong) [inline spec/helpers/middleman/page_spec.rb:195]: Oh ye this is where the test is failing - title mismatches. Might be fine to just delete these though as Kylor mentioned that the test coverage here isn't that important... up to you!

Comment (maisiembleton) [reply spec/helpers/middleman/page_spec.rb]: awesome thanks! might as well remove them if they're not needed

File: spec/helpers/middleman/page_spec.rb (hunk 1 above)
Comment (maisiembleton) [reply spec/helpers/middleman/page_spec.rb]: sweet - removed them !

**Pull request merged**